src/index.js

```diff
@@ -4,13 +4,25 @@
 
 // Koa application to provide HTTP interface.
 
+const fs = require("fs");
+const envPaths = require("env-paths");
+const config_file = envPaths("oasis", { suffix: "" }).config + "/config.json";
+console.log(`Trying to read config file ${config_file}`);
+var presets;
+if (fs.existsSync(config_file)) {
+  presets = require(config_file);
+} else {
+  presets = {};
+}
 const cli = require("./cli");
-const config = cli();
+const config = cli(presets);
 
 if (config.debug) {
   process.env.DEBUG = "oasis,oasis:*";
 }
 
+console.log(JSON.parse(JSON.stringify(config)));
```

Comment:
I'm concerned that this is going to print private keys -- could you confirm that isn't the case before we merge? Sorry, I hate to nitpick but I'm afraid someone will copy and paste the startup output and accidentally leak their private key somewhere.

Reply:
valid concern, but this is the oasis config file, not the config from `~/.ssb`. Is there a mechanism that combines the two somehow?
On a code level, that line is in `index.js` and is executed way before we even `require("./ssb");`
For me, it prints this:

```js
{ _: [],
  open: false,
  offline: false,
  host: 'localhost',
  port: 13132,
  public: false,
  debug: true,
  '$0': 'oasis' }
```

I'd be happy to figure out how to get `_` and `$0` out of there, but I don't see how the ssb config would be printed here, hence no key material.

Reply:
Oops, you're totally right. Thanks for reminding me how that worked!